shell/ios/ViewController.mm

```diff
@@ -130,7 +130,7 @@ - (void)initRenderSessionController {
     IGL_DEBUG_ASSERT(platformDevice);
     return igl::SurfaceTextures{
         .color = platformDevice->createTextureFromNativeDrawable((CAEAGLLayer*)layer_, nullptr),
-        .depth = platformDevice->createTextureFromNativeDepth((CAEAGLLayer*)layer_, igl::TextureFormat::Z_UNorm16, nullptr),
+        .depth = platformDevice->createTextureFromNativeDepth((CAEAGLLayer*)layer_, igl::TextureFormat::S8_UInt_Z24_UNorm, nullptr),
```

Comment:
This should be controlled from the app side. The default format should remain `Z_UNorm16`.